no-computer.md

```diff
@@ -10,41 +10,82 @@ good way to meet their needs.
+game. Many abstract strategy games require a lot of time, but this game is pretty 
+short and fun to play in your spare time. 
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
+In this game, I would consider the different colored pieces as the vocabulary
+of the DSL, and the different positions on the board as the governing syntax. 
+The DSL is necessary for the players to communicate their moves and to become
+closer to reaching an end state of the game. They use the language by swapping 
+one piece with a piece in a different position on the circle or in the center of
+the circle. The "code" is modified during each player's turn when they must swap
+a piece. 
 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
+This idea excites me because I'm looking forward to play testing it with others
+and seeing their reactions. It's nice to make something that brings out a 
+competitive spirit in people and allows them to have some fun. I came up with it
+while brainstorming ideas with some of my peers in my game design class.
 
 
 ### Domain
 _Describe the project's domain in five words._
+Two-player, abstract strategy board games
 
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
+The user interacts with the language by swapping their colored pieces with
+other pieces on the board. They observe the entire board at each turn, and 
+they try to pick the best move by reasoning through the outcomes of different
+swaps. Involving the whole board throughout the entire game is an intentional
+choice that forces the user to be thinking about many different moves each
+turn. The language environment is complex, even though the vocabulary of the
+language is quite simple.
 
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
+When the game is in play, the position of the colored pieces indicates the state
+of the game to the user. The game state changes with each player's turn. When the 
+user has many pieces lined up across from each other, it indicates to the them that 
+they are close to winning. If a user makes an illegal move, it is up to their 
+opponent to call them out! 
 
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
+It is easy to swap colored pieces in this language and easy to observe the entire
+state of the game, or "program". It is difficult to make "sneaky" changes in this
+language, since the user's opponent can always observe the moves they are making.
+It is impossible to create a computer program with this language. :)
```

Comment:
But it's very possible to code this in a way that it becomes a game that someone can play on the computer! :)